test/lib/tasks/minify.js

```diff
@@ -148,3 +152,94 @@ test();`;
 		"Third taskUtil.setTag call with expected arguments");
 });
 
+test("integration: minify (without taskUtil)", async (t) => {
+	const {reader, writer, workspace} = createWorkspace();
+	const content = `
+function test(paramA) {
```

Comment:
should we add there some indentation?

Reply:
answered above https://github.com/SAP/ui5-builder/pull/709#discussion_r823602439